readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://1stIndominus.github.io/layout_moyo-header/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/170074786-65657408-682a-4975-bb18-3c89738ae2db.png)
Header should take all available space